### ISSUE TITLE
Run CLI `stylelint` without params should output help

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1557]
If i run in cli:
```shell
stylelint
```
Should output help (`stylelint -h`)

**Comment by jeddy3** [comment]
If this is standard practice then OK by me.

**Comment by evilebottnawi** [comment]
@jeddy3 
```
eslint
```
Show help, @davidtheclark that do you think?

**Comment by davidtheclark** [comment]
That seems weird to me. Do other CLIs do this, besides ESLint?

**Comment by evilebottnawi** [comment]
@davidtheclark yes `webpack`, `postcss-cli`, `npm-run-all` and many other.

**Comment by davidtheclark** [comment]
Ok, sounds good to me.


On June 29, 2016 at 11:17:49 AM, m-allanson (notifications@github.com)
wrote:

I think it's quite common behaviour for CLI programs that require options
to run. Picking a few examples from the node_modules/.bin/ folder of my
current project: jspm, webpack, svgo and traceur all exhibit similar
behaviour. Also try cp, ssh or mv in a terminal to see shell examples.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/stylelint/stylelint/issues/1557#issuecomment-229442657>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAmWz0dckI0EfwcI-Tab--VBRdmdgy1Uks5qQrbMgaJpZM4JBT9S>
.
